packaging/centos-7/build-image/Dockerfile

```diff
@@ -0,0 +1,33 @@
+FROM alanfranz/fwd-centos-7:latest
+MAINTAINER support@osuosl.org
+RUN yum -y install \
+    gcc \
+    gcc-c++ \
+    glibc \
+    glibc-devel \
+    glibc-static \
+    libstdc++-static \
+    make \
+    tar \
+    wget \
+    zlib \
+    zlib-devel \
+    zlib-static \
+    && yum clean all
+RUN mkdir /dssp
+WORKDIR /dssp
+RUN wget http://www.bzip.org/1.0.6/bzip2-1.0.6.tar.gz && \
+    tar zxf bzip2-1.0.6.tar.gz
```

Comment:
Why do we need to install bzip manually like this? Why can't you use bzip2-devel?